common/presenters/single-requests-to-table-component.js

```diff
@@ -24,6 +28,7 @@ function singleRequestsToTable({ isSortable = true, query } = {}) {
             const card = moveToCardComponent({
               isCompact: true,
```

Comment:
I think this is effectively the option that will tell `profileToCardComponent` to not show any of the metadata so we can drop needing to include `locationType` in this presenter.